docs/source/sandbox/packages.rst

```diff
@@ -67,48 +67,111 @@ an experiment. Then to "create" an environment, ReproNim needs to reinstall the
 packages from the specification (ideally matching as many properties, such as
 version, architecture, size, and hash as possible).
 
-Known distributions
--------------------
-we might want to cover, and underlying distribution "toolkits"
+Package Management and Environment Configuration
+------------------------------------------------
 
-GNU/Linux (Core? OS?)
-~~~~~~~~~~~~~~~~~~~~~
-- Debian - dpkg, apt
-- Ubuntu - dpkg, apt
-- CentOS - rpm, yum
+Here we discuss package managers and key distributions that ReproNim should
+cover (and list other potential package managers to consider)
+
+OS Package Managers
+~~~~~~~~~~~~~~~~~~~
+
+- apt-get (dpkg) - Expected on Debian and Ubuntu Gnu/Linux distributions
+- yum (rpm) - Expected on CentOS/RHEL and other Red Hat Gnu/Linux distributions
+- snap - Linux packages (with sandboxed execution) - http://snapcraft.io/
 
-and additional "overlays" in terms of APT repositories
+  - Snaps may prove difficult for tracing because commands to download
+    and build executibles can be embedded into snap packages
```

Comment:
Snaps that have just a compressed archive you will be able to query.  But for those that download an archive and then run a script to build it, I don't see how you can easily query what files those contributed (unless you installed the snap in a sandbox and figured out what it created...).